examples/langchain/README.md

```diff
@@ -147,3 +149,18 @@ into different collections.
 ```
 python load_data_with_embeddings.py
 ```
+
+### Running the Vector Query
+
+You are now ready to test the example vector retriever. Run the following to ask a
+question with the results augmented via the `marklogic_vector_query_retriever.py` module
+in this project:
+
+    python ask_vector_query.py "What is task decomposition?" posts_with_embeddings
+
+This retriever uses a CTS search for retrieving candidate documents, and defaults to
```

Comment:
Not important right now, but ultimately, I think we only want to say "This retriever searches MarkLogic for candidate documents" - i.e. some / many users will have no idea what "CTS search" means.